Watchdog/Watchdog.ino

```diff
@@ -1,17 +1,17 @@
-int inPin = 7; //whatever the input pin is
-int outPin = 13; //output to tell other hardware if software failed
+int inPin = 1; //whatever the input pin is
+int outPin = 0; //output to tell other hardware if software failed
 int heartBeat = 0;   //output value, start LOW
 int lastBeat = 0; //to check for changing vals
-int changeCounter = 0; //just looking for changes
-int totalCounter = 0; //to check if we are changing fast enough
-int mainLoop = 0; //turn on once we know program is running
 
 //wait for X amount of signal changes (heart beats) within a short 
 //time period (to make sure power cycles or other transients doesn't 
 //accidentally turn it on)
 //no idea what these should be
-int waitPeriod = 15;
-int goal = 10; 
+int changeCounter = 0; //just looking for changes
+int waitPeriod = 260;
```

Comment:
Set all of these values to `0`.

Create a function called `setupState0()` that sets all the variables to their correct values for state 0.